qiita_pet/handlers/api_proxy/tests/test_studies.py

```diff
@@ -52,28 +52,79 @@ def test_study_get_req(self):
                     'Future studies will attempt to analyze the soils and '
                     'rhizospheres from the same location at different time '
                     'points in the plant lifecycle.',
-                    'status': 'private',
-                    'spatial_series': False,
-                    'study_description': 'Analysis of the Cannabis Plant '
-                                         'Microbiome',
-                    'shared_with': ['shared@foo.bar'],
-                    'lab_person': {'affiliation': 'knight lab',
-                                   'name': 'LabDude',
-                                   'email': 'lab_dude@foo.bar'},
-                    'principal_investigator': {'affiliation': 'Wash U',
-                                               'name': 'PIDude',
-                                               'email': 'PI_dude@foo.bar'},
-                    'study_alias': 'Cannabis Soils',
-                    'study_id': 1,
-                    'most_recent_contact': datetime(2014, 5, 19, 16, 11),
-                    'publications': [['10.100/123456', '123456'],
-                                     ['10.100/7891011', '7891011']],
-                    'num_samples': 27,
-                    'study_title': 'Identification of the Microbiomes for '
-                                   'Cannabis Soils',
-                    'number_samples_collected': 27}}
+                'status': 'private',
```

Comment:
The changes here are just fixing the indentation (PEP8 didn't flag it but it was incorrect).